### DISCUSSION TITLE
Issue with Isaac Sim example with iiwa_allegro.yml

### DISCUSSION THREAD

**Discussion by masaaldosey** [NVlabs/curobo#140] [Software Q&A] [answered]
- I am trying to run the `motion_gen_reacher.py` example with the `iiwa_allegro.yml` robot configuration file.
- I have not made any changes to the `iiwa_allegro.yml` file.
- When I click the _Play_ button and move the _target cube_ around, I always run into `[Warning] [__main__] Plan did not converge to a solution.  No action is being taken.`
- Could you please help me solve this? My software versions are - [cuRobo hash](c09d94908d8ad52f557ed59195c1ceb2d0434d65), Isaac Sim 2022.2.1.
- I looked at discussions:
   - #121 
   - #81

https://github.com/NVlabs/curobo/assets/48513587/6bd99e21-426a-46a3-9978-c6d99fc39508

**Comment by balakumar-s** [comment, marked as answer]
iiwa_allegro is meant to be run with multi_arm_reacher.py example as the configuration file for the robot tries to track multiple fingertips. 

If you want to use it for single target reaching, make `link_names: null` here https://github.com/NVlabs/curobo/blob/88eac64edca2d2b641c1622e63f3e750157596c4/src/curobo/content/configs/robot/iiwa_allegro.yml#L36

**Reply by masaaldosey** [reply]
Thank you, it works now! However, would it be possible to skip the `ee_link` altogether and send directly the fingertips?

**Reply by balakumar-s** [reply]
You can make ee_link: one of the fingertips. That will make all targets be for the fingertips.
